src/classes/AffiliationMappingsHealthCheckVMapper.cls

```diff
@@ -0,0 +1,314 @@
+        
+        childAffiliationHCGroupVModelsList.add(this.getValidRecordTypeHealthCheckGroupVModel(
+            affiliationMappingsModelsList, 
+            accRTModelsById
+        ));
+
+        childAffiliationHCGroupVModelsList.add(this.getValidContactPrimaryFieldHealthCheckGroupVModel(
+            affiliationMappingsModelsList, 
+            accountLookupsOnContactFieldModelsByName,
+            accountLookupsOnContactFieldModelsByLabel
+        ));
+
+        return this.combineAffiliationMappingsHealthCheckGroupViewModels(childAffiliationHCGroupVModelsList);
+    }
+
+    /**********************************************************************************
+    * @description Generates a HealthCheckGroupVModel comprised of the combined health 
+    * check results for Default Account, Administrative and Household accounts. 
+    * @param  healthCheckGroupVModelsList A list of HealthCheckGroupVModels
+    * @return A HealthCheckGroupVModel representing the health check results for 
+    * Default, Administrative and Household Accounts.
+    **********************************************************************************/ 
+    @TestVisible
+    private HealthCheckGroupVModel combineAffiliationMappingsHealthCheckGroupViewModels(List<HealthCheckGroupVModel> healthCheckGroupVModelsList) {
+        HealthCheckGroupVModel parentHealthCheckGroupVModel = 
+            new HealthCheckGroupVModel(
+                Label.stgHealthCheckAffiliationMappings,
+                Label.stgHealthCheckStatusPassed,
+                0,
+                0,
+                new List<String>(),
+                new List<HealthCheckItemVModel>()
+            );
+
+        for (HealthCheckGroupVModel healthCheckGroupVModel : healthCheckGroupVModelsList) {
+            parentHealthCheckGroupVModel.passedChecks += healthCheckGroupVModel.passedChecks;
+            parentHealthCheckGroupVModel.totalChecks += healthCheckGroupVModel.totalChecks;
+
+            parentHealthCheckGroupVModel.expandedRowsList.addAll(healthCheckGroupVModel.expandedRowsList);
+            parentHealthCheckGroupVModel.healthCheckItemList.addAll(healthCheckGroupVModel.healthCheckItemList);
+        }
+
+        parentHealthCheckGroupVModel.updateHealthCheckStatus();
+        parentHealthCheckGroupVModel.healthCheckItemList.sort();
+        
+        return parentHealthCheckGroupVModel;
+    }
+
+    /**********************************************************************************
+    * @description Retrieve a HealthCheckGroup view model reprenting whether primary
+    * affiliation fields specified in affiliation mappings correspond to valid fields
+    * on the Contact object that lookup to Account objects.
+    * @return A HealthCheckGroupVModel
+    **********************************************************************************/ 
+    @TestVisible 
+    private virtual HealthCheckGroupVModel getValidContactPrimaryFieldHealthCheckGroupVModel(
```

Comment:
Just my opinion, should we follow the same order of how these methods been used?